xtask/src/main.rs

```diff
@@ -25,40 +24,31 @@ use linux as plat;
 
 // Possible formats for a bundled dendrite distro.  Currently the two "zone"
 // package formats are helios-only.
-#[derive(PartialEq, Debug)]
+#[derive(PartialEq, Clone, Debug, ValueEnum)]
 pub enum DistFormat {
-    Native,  // .deb or .p5p, depending on the platform
-    Omicron, // package to be included in an omicron zone
-    Global,  // package to run standalone in the global zone
+    /// .deb or .p5p, depending on the platform
+    #[value(alias = "n")]
```

Comment:
The `ValueEnum` means we get the `--format omicron` for free without the `FromStr` implementation, right?  The buildomat jobs are still using the long form of the option and passing, so I assume this is fine.

Reply:
Yeah, both the short and long forms will work, so there aren't any behavior changes compared to today.